schema/stake.proto

```diff
@@ -21,10 +22,10 @@ message FindStakeRequest {
 }
 
 message FindStakeResponse {
-    repeated Stake stakes = 1;
+    StakeDetails stake = 1;
```

Comment:
Is this still used?

AFAIK it's been deprecated in favour of https://github.com/dusk-network/rusk/blob/5e9b176936efd6601874c1d00434188085bd90cb/schema/state.proto#L87